392 swa

armin-vcg opened the pull request:
Script works for reference, but FYI, a tag mistake has 2.0.1-SNAPSHOT in the POM.  The script works from the tag, so generates directories / files with the snapshot name.
README is done, unless changes requested.

abhijitvk commented on local-dev/swa/README.md:
```diff
@@ -0,0 +1,55 @@
+	* A zip of the local Fortify rulepack that used to scan the code
+
+## When and how do I run the script?
+
```
@armin-vcg : I think we must add pre-requisites to run the script. may be something as below?
1) Installation of Fortify SCA and Apps, the path as configured in defaults/swa-prep.properties file
2) Instructions and default values are tested on MAC

armin-vcg replied:
Added prerequisites section, with instructions. Will show when I push all changes requested.

abhijitvk commented:
<img width="722" alt="Screen Shot 2019-07-12 at 12 26 19 PM" src="https://user-images.githubusercontent.com/11605414/61143498-648a0400-a4a0-11e9-9dc5-6c9405379621.png">

@armin-vcg  - Confused with this line `Review files and folders under /Applications/Fortify/Fortify_SCA_and_Apps_18.20/Core/config`. What are the files under this folder to be reviewed?

Also, just making sure I expect the output folder as below. so essentially the final step would be to create a package as a zip file from under folder `submission-files` 

<img width="1087" alt="Screen Shot 2019-07-12 at 12 28 58 PM" src="https://user-images.githubusercontent.com/11605414/61143605-aa46cc80-a4a0-11e9-9020-2ef0994cf947.png">

abhijitvk commented:
<img width="1369" alt="Screen Shot 2019-07-12 at 1 03 52 PM" src="https://user-images.githubusercontent.com/11605414/61145491-9487d600-a4a5-11e9-9bd2-e4ea403ea55b.png">

Build label shows as `bip-reference-perftest-0.0.1-SNAPSHOT`. Is this correct? Just wondering why it didn't label as `bip-reference-reactor-0.0.1-SNAPSHOT`

armin-vcg commented:
Ready for another round of review.
- Removed dependency on parentpom project (have not deleted the project yet)
- The "fortify" and "fortify-sca" maven profiles coexist, added "fortify-merge" profile
- Added fortify.sh in root folder
- Fixed swa-prep.sh to use new maven profiles, guarantee clean "source zip"
- Updated docs
